src/IO/CMakeLists.txt

```diff
@@ -24,9 +24,11 @@ add_spectre_library(${LIBRARY} ${LIBRARY_SOURCES})
 
 target_link_libraries(
   ${LIBRARY}
-  INTERFACE DataStructures
-  INTERFACE ErrorHandling
-  INTERFACE Utilities
+  PUBLIC
```

Comment:
From what I've read in the links you sent me, it seems like `Hdf5` and `ErrorHandling` should be private. `Utilities` in theory should be private, but I think because `TaggedTuple` is still there it might not be.

Reply:
Yea I would like to make `Hdf5` and `ErrorHandling` private, but some header files in `IO` include headers from those libraries and therefore they must probably be public, don't you think?

Reply:
Oh yea, you're right... We can see if that can be fixed at some point in the future :)